submissions/andrewklmn/html-css-popup/app.css

```diff
@@ -0,0 +1,253 @@
+* {
+  padding: 0px;
+  margin: 0px;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  text-align: center;
+  /*
+  background-image: url(img/bg_image.jpg);
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center; 
+  */
+  width: 100vw;  
+  height: 100vh;
+}
+
+label.button {
+  color:black;
+  cursor: pointer;
+  border: white solid thin;
+  box-sizing: border-box;
+}
```

Comment:
Such selectors are difficult to change in the future due to their weight. It is better to keep the weight of the selector to a minimum, for example, just by class.

U can read about it https://css-tricks.com/specifics-on-css-specificity/